### PR TITLE
test: add install/uninstall deployment script test

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
test: add install/uninstall deployment script test
This PR is mainly for testing scripts:
 - deploy/install-driver.sh
 - deploy/uninstall-driver.sh

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by andyzhangx** [comment]
The script deployment test works well, we will use this test to make sure `deploy/install-driver.sh`, `deploy/uninstall-driver.sh` works well since most users would use these scripts.

```
2020/03/31 03:40:07 Azure Disk CSI Driver uninstalled
2020/03/31 03:40:07 ===================install Azure Disk CSI Driver deployment scripts test===================
use local deploy
Installing Azure Disk CSI driver, version: master ...
csidriver.storage.k8s.io/disk.csi.azure.com created
customresourcedefinition.apiextensions.k8s.io/csinodeinfos.csi.storage.k8s.io created
serviceaccount/csi-azuredisk-controller-sa created
clusterrole.rbac.authorization.k8s.io/azuredisk-external-provisioner-role created
clusterrolebinding.rbac.authorization.k8s.io/azuredisk-csi-provisioner-binding created
clusterrole.rbac.authorization.k8s.io/azuredisk-external-attacher-role created
clusterrolebinding.rbac.authorization.k8s.io/azuredisk-csi-attacher-binding created
clusterrole.rbac.authorization.k8s.io/azuredisk-cluster-driver-registrar-role created
clusterrolebinding.rbac.authorization.k8s.io/azuredisk-csi-driver-registrar-binding created
clusterrole.rbac.authorization.k8s.io/azuredisk-external-snapshotter-role created
clusterrolebinding.rbac.authorization.k8s.io/azuredisk-csi-snapshotter-binding created
clusterrole.rbac.authorization.k8s.io/azuredisk-external-resizer-role created
clusterrolebinding.rbac.authorization.k8s.io/azuredisk-csi-resizer-role created
deployment.apps/csi-azuredisk-controller created
daemonset.apps/csi-azuredisk-node created
install snapshot driver ...
customresourcedefinition.apiextensions.k8s.io/volumesnapshots.snapshot.storage.k8s.io created
customresourcedefinition.apiextensions.k8s.io/volumesnapshotclasses.snapshot.storage.k8s.io created
customresourcedefinition.apiextensions.k8s.io/volumesnapshotcontents.snapshot.storage.k8s.io created
serviceaccount/csi-snapshot-controller-sa created
clusterrole.rbac.authorization.k8s.io/csi-snapshot-controller-role created
clusterrolebinding.rbac.authorization.k8s.io/csi-snapshot-controller-binding created
clusterrole.rbac.authorization.k8s.io/csi-snapshot-controller-leaderelection-role created
clusterrolebinding.rbac.authorization.k8s.io/csi-snapshot-controller-leaderelection-binding created
deployment.apps/csi-snapshot-controller created
install Windows driver ...
daemonset.apps/csi-azuredisk-node-win created
Azure Disk CSI driver installed successfully.
2020/03/31 03:40:14 ===================================================
2020/03/31 03:40:14 ===================uninstall Azure Disk CSI Driver deployment scripts test===================
use local deploy
Uninstalling Azure Disk CSI driver, version: master ...
customresourcedefinition.apiextensions.k8s.io "volumesnapshots.snapshot.storage.k8s.io" deleted
customresourcedefinition.apiextensions.k8s.io "volumesnapshotclasses.snapshot.storage.k8s.io" deleted
customresourcedefinition.apiextensions.k8s.io "volumesnapshotcontents.snapshot.storage.k8s.io" deleted
serviceaccount "csi-snapshot-controller-sa" deleted
clusterrole.rbac.authorization.k8s.io "csi-snapshot-controller-role" deleted
clusterrolebinding.rbac.authorization.k8s.io "csi-snapshot-controller-binding" deleted
clusterrole.rbac.authorization.k8s.io "csi-snapshot-controller-leaderelection-role" deleted
clusterrolebinding.rbac.authorization.k8s.io "csi-snapshot-controller-leaderelection-binding" deleted
deployment.apps "csi-snapshot-controller" deleted
deployment.apps "csi-azuredisk-controller" deleted
daemonset.apps "csi-azuredisk-node" deleted
daemonset.apps "csi-azuredisk-node-win" deleted
csidriver.storage.k8s.io "disk.csi.azure.com" deleted
customresourcedefinition.apiextensions.k8s.io "csinodeinfos.csi.storage.k8s.io" deleted
serviceaccount "csi-azuredisk-controller-sa" deleted
clusterrole.rbac.authorization.k8s.io "azuredisk-external-provisioner-role" deleted
clusterrolebinding.rbac.authorization.k8s.io "azuredisk-csi-provisioner-binding" deleted
clusterrole.rbac.authorization.k8s.io "azuredisk-external-attacher-role" deleted
clusterrolebinding.rbac.authorization.k8s.io "azuredisk-csi-attacher-binding" deleted
clusterrole.rbac.authorization.k8s.io "azuredisk-cluster-driver-registrar-role" deleted
clusterrolebinding.rbac.authorization.k8s.io "azuredisk-csi-driver-registrar-binding" deleted
clusterrole.rbac.authorization.k8s.io "azuredisk-external-snapshotter-role" deleted
clusterrolebinding.rbac.authorization.k8s.io "azuredisk-csi-snapshotter-binding" deleted
clusterrole.rbac.authorization.k8s.io "azuredisk-external-resizer-role" deleted
clusterrolebinding.rbac.authorization.k8s.io "azuredisk-csi-resizer-role" deleted
Uninstalled Azure Disk CSI driver successfully.
2020/03/31 03:40:18 ===================================================
```

File: test/e2e/suite_test.go [hunk 1]
```diff
@@ -118,6 +118,22 @@ var _ = ginkgo.AfterSuite(func() {
...
 			}
 			execTestCmd([]testCmd{azurediskLog, e2eTeardown})
+
+			// install/uninstall Azure Disk CSI Driver deployment scripts test
+			installDriver := testCmd{
+				command:  "bash",
+				args:     []string{"deploy/install-driver.sh", "master", "windows,snapshot,local"},
+				startLog: "===================install Azure Disk CSI Driver deployment scripts test===================",
+				endLog:   "===================================================",
+			}
```
Comment (ZeroMagic) [inline test/e2e/suite_test.go]: Is it enough that we just test the master branch image here? The image does not include this PR.

Comment (andyzhangx) [reply test/e2e/suite_test.go]: we only test the deployment script here. Original e2e tests already tested against the image.

**Pull request merged**